code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Core.dm

```diff
@@ -170,10 +170,10 @@
 		remove_self(needed)
 
 /datum/reagent/water/affect_blood(var/mob/living/carbon/M, var/alien, var/removed)
-	if(alien == IS_SLIME)
+	/*if(alien == IS_SLIME) //VOREStation Edit Start. Stops slimes from dying from water.
 		M.adjustToxLoss(6 * removed)
-	else
-		..()
+	else*/
+	..() //VOREStation Edit End.
 
 /datum/reagent/fuel/affect_ingest(var/mob/living/carbon/M, var/alien, var/removed)
```

Comment:
There's a PR upstream to change this 'cause they copypasted and put "fuel" in instead of water. Just comment out both procs entirely, 'cause this probably breaks fuel's _actual_ affect_ingest() too.

Reply:
Resolved. Did a /* */ above water and below fuel, since they're both unneeded/borked

Reply:
Fuel doesn't have an affect_ingest(), so there would be nothing to mess up. It would make prometheans take more damage from drinking fuel (which would be fine, really). But like mentioned, it's already fixed upstream to water.